packages/Accordion/package.json

```diff
@@ -5,6 +5,10 @@
     "main": "dist/cjs/Accordion.js",
     "module": "dist/esm/Accordion.js",
     "types": "dist/cjs/index.d.ts",
+    "files": [
+        "dist/cjs/styles.css",
+        "dist/cjs/styles.css"
```

Comment:
Twice?

Reply:
oops that's suppose to be "dist/esm/styles.scs" for the second one. let me fix that

Reply:
updated